tests/test_hardware.py

```diff
@@ -7,19 +7,24 @@
 
 import unittest
 from unittest import TestCase
-
 from collections import OrderedDict
 
-from ._helpers import create_outdir, mpi_multi
-
 from sotodlib.core import Hardware
+from sotodlib.sim_hardware import sim_nominal
+from sotodlib.vis_hardware import plot_detectors
 
-from sotodlib.sim_hardware import get_example
+from ._helpers import create_outdir, mpi_multi
 
-from sotodlib.sim_hardware import (sim_wafer_detectors,
-                                   sim_telescope_detectors)
 
-from sotodlib.vis_hardware import plot_detectors
+toast_available = None
```

Comment:
I think we have discussed this import guard before. Since you set `toast_available` to `None` and immediately check its value, the test is redundant and can be omitted. Furthermore, because of Python scoping rules, you don't even need to declare `toast_available` before the `try-except` clause.

Reply:
Good point, I removed that.  Also, in the `toast3_data` PR, I moved this check into the `_helpers.py` file for use in multiple tests.